### ISSUE TITLE
structor file is missing after updating to version 1.4.0

### ISSUE THREAD

**Issue by farodin91** [containous/structor#9]
My command is 
```
structor -o farodin91 -r janusgraph \
            --dockerfile-url="https://raw.githubusercontent.com/farodin91/janusgraph/v0.2/docs.Dockerfile" \
            --menu.js-url="https://raw.githubusercontent.com/farodin91/janusgraph/v0.2/docs/theme/js/structor-menu.js.gotmpl" \
            --exp-branch=mkdocs --debug;
```
Build: https://travis-ci.com/farodin91/janusgraph/jobs/179779504
gh-pages: https://github.com/farodin91/janusgraph/tree/gh-pages
branch: https://github.com/farodin91/janusgraph/tree/v0.2

**Comment by farodin91** [comment]
The logs doesn't help me.

**Comment by ldez** [comment]
Do you mean that the site files are not generated?

**Comment by farodin91** [comment]
menu file is missing.

**Comment by farodin91** [comment]
I'm trying currently a differnt structor menu file.

**Comment by ldez** [comment]
Maybe it's related to bin owner on your CI :thinking:

**Comment by farodin91** [comment]
Here is the structor-menu file missing.
```yaml
      before_deploy:
        - echo "Download documentation generator";
          curl -sfL https://raw.githubusercontent.com/containous/structor/master/godownloader.sh | bash -s -- -b $GOPATH/bin v1.4.0
        - echo "Build documentation";
          structor -o farodin91 -r janusgraph \
            --dockerfile-url="https://raw.githubusercontent.com/farodin91/janusgraph/v0.2/docs.Dockerfile" \
            --menu.js-url="https://raw.githubusercontent.com/farodin91/janusgraph/v0.2/docs/theme/js/structor-menu.js.gotmpl" \
            --exp-branch=mkdocs --debug;
```

Here exists the structor-menu file.
```yaml
      before_deploy:
        - echo "Download documentation generator";
          echo https://github.com/containous/structor/releases/download/v1.3.2/structor_linux-amd64 | wget --output-document=$GOPATH/bin/structor -i -;
          chmod +x $GOPATH/bin/structor;
        - echo "Build documentation";
          structor -o farodin91 -r janusgraph \
            --dockerfile-url="https://raw.githubusercontent.com/farodin91/janusgraph/v0.2/docs.Dockerfile" \
            --menu.js-url="https://raw.githubusercontent.com/farodin91/janusgraph/v0.2/docs/theme/js/structor-menu.js.gotmpl" \
            --rqts-url="https://raw.githubusercontent.com/containous/structor/master/requirements-override.txt" \
            --exp-branch=mkdocs --debug;
```

**Comment by ldez** [comment]
Have you try locally, on your computer?

**Comment by farodin91** [comment]
I will try it.

**Comment by farodin91** [comment]
Same problem on my computer with 1.4.0.

**Comment by ldez** [comment]
I think I found the issue.

**Comment by ldez** [comment]
I found the issue, thanks to @dduportal :tada:

You can use v1.4.1

**Comment by ldez** [comment]
sorry I made a mistake in the 1.4.1.
v1.4.2 coming soon.

**Comment by ldez** [comment]
Fixed in v1.4.2 :tada:

**Comment by farodin91** [comment]
@ldez Thank you. 
Tested and verified.
